packages/storage/src/uploads.ts

```diff
@@ -66,7 +60,7 @@ export const uploadFile = async (
   file: File,
 ): Promise<ArweaveResponse> => {
   if (MINTBASE_API_KEY == MINTBASE_API_ANON_USER) {
-    console.warn(ANON_USER_WARNING);
+    console.warn('Warning: you are using the anonymous mintbase api key. You need to specify yours in order to upload a file.');
```

Comment:
Let's leave this warning, but say "in the future, you will need to obtain a session key from mintbase connect"

Reply:
Good idea